Mesh_3/test/Mesh_3/test_meshing_without_features_determinism.cpp

```diff
@@ -0,0 +1,157 @@
+#include <CGAL/Exact_predicates_inexact_constructions_kernel.h>
+#include <CGAL/Mesh_triangulation_3.h>
+#include <CGAL/Mesh_complex_3_in_triangulation_3.h>
+#include <CGAL/Mesh_criteria_3.h>
+
+#include <CGAL/Polyhedral_mesh_domain_with_features_3.h>
```

Comment:
This can be replaced by `Polyhedra_mesh_domain_3.h` to be entirely out of the "with features" framework